docs: Add AI/ML API example

D1m7asis opened the pull request:
### 🔄 Summary

This PR adds support for [AI/ML API](https://aimlapi.com/app/), an OpenAI-compatible provider offering 300+ models (e.g., DeepSeek, Gemini, ChatGPT), to the E2B Code Interpreter examples.

---

### ✅ What's Included

* 🧪 examples/aimlapi-js/: TypeScript example using E2B sandbox + AI/ML API via OpenAI SDK
* 🧪 examples/aimlapi-python/: Python version using openai and e2b-code-interpreter
* 📊 Uses a CSV dataset to generate and run Python code (e.g., linear regression)
* 📄 Added .env.template, README.md, and dependency setup for each variant
* 🔗 Linked AI/ML API to the providers table in root README.md with proper UTM tracking

---

Let me know if you'd like the examples styled/formatted differently — happy to iterate!

Closes #106

D1m7asis committed:
Add AI/ML API example

D1m7asis commented:
@mishushakov hi! Can i get a review here pls? Thanks!

mishushakov committed:
Merge branch 'main' into main

mishushakov reviewed (changes requested):
Hey @D1m7asis, some points:

1. I have tried running the JS example with my own API key and got this:

    ```js
    ❌ Error: 2 | import { castToError } from "./core.mjs";
    3 | export class OpenAIError extends Error {
    4 | }
    5 | export class APIError extends OpenAIError {
    6 |     constructor(status, error, message, headers) {
    7 |         super(`${APIError.makeMessage(status, error, message)}`);
                              ^
    error: 403 status code (no body)
    ```

    What am I doing wrong?

2. Can you make sure the example is same for both JS and Python?

3. I see that the examples are using `gpt-3.5-turbo` - can you use a more recent model, as not a lot of people would be starting with GPT-3.5 today

D1m7asis committed:
Enhance AIML API examples

Improved the JavaScript and Python AIML API example scripts for clarity, modularity, and robustness. Updated documentation for both examples, clarified model usage, and added a sample data.csv file for reproducible data science tasks. The JS and Python scripts now better extract and execute code from LLM responses, handle datasets, and save generated images.

D1m7asis committed:
Merge branch 'main' of https://github.com/D1m7asis/e2b-cookbook-aimlapi

mishushakov reviewed (changes requested):
left some comments regarding getting the stdout of the sandbox

D1m7asis committed:
Simplify sandbox code execution and output handling

Refactored JS and Python examples to streamline code execution in sandboxes, removing manual stdout/stderr collation and using the first result's text and image directly. Updated .env.example to include AIML_API_KEY and fixed README to link to the TypeScript example for AIML API.

D1m7asis commented:
@mishushakov thanks for the review 🙌
Before making the changes, I’d like to check, does e2b have a Slack for integration discussions?
If yes, could you please share an invite/connect to sergey@aimlapi.com (Sergey Nuzhny, DevRel + CM at AIMLAPI).
We can follow up with more details there if needed.

D1m7asis committed:
Update README.md

D1m7asis committed:
Update README.md

D1m7asis committed:
Merge branch 'main' into main
